babel.config.js

```diff
@@ -93,7 +93,7 @@ module.exports = function getBabelConfig(api) {
       {
         useESModules,
         // any package needs to declare 7.25.0 as a runtime dependency. default is ^7.0.0
-        version: '^7.25.0',
+        version: process.env.MUI_BABEL_RUNTIME_VERSION || '^7.25.0',
```

Comment:
Not 100% satisfied with this solution but the babel config is used both by our build and by `babel-node`. `MUI_BABEL_RUNTIME_VERSION` may not always be defined. This seemed to be the lesser of evils, the existence is already checked in the build script